packages/terra-clinical-item-display/src/ItemDisplay.scss

```diff
@@ -39,6 +39,14 @@
     font-weight: 700 !important;
```

Comment:
Team: Why do these have `!important`? This seems potentially restrictive to consumers.

Reply:
I don't see any reason why it would be needed. My vote would be to remove it.